# Available commands to use the Fisher Scientific PPS4102 Top Pan balance 
int32 HEAT_ON = 0
int32 HEAT_OFF = 1
int32 STIR_ON = 2
int32 STIR_OFF = 3
int32 SET_STIR = 4
int32 SET_HEAT = 5
int32 STIR_AT = 6
int32 HEAT_AT = 7
int32 ALL_OFF = 8

int32 ika_command
int32 ika_param
